src/server/status_server.rs

```diff
@@ -210,6 +212,142 @@ impl StatusServer {
         Box::new(ok(res))
     }
 
+    fn frames_post_processor() -> impl Fn(&mut pprof::Frames) {
+        let thread_rename = [
+            (Regex::new(r"^grpc-server-\d*$").unwrap(), "grpc-server"),
+            (Regex::new(r"^cop-high\d*$").unwrap(), "cop-high"),
+            (Regex::new(r"^cop-normal\d*$").unwrap(), "cop-normal"),
+            (Regex::new(r"^cop-low\d*$").unwrap(), "cop-low"),
+            (Regex::new(r"^raftstore-\d*$").unwrap(), "raftstore"),
+            (Regex::new(r"^raftstore-\d*-\d*$").unwrap(), "raftstore"),
+            (Regex::new(r"^sst-importer\d*$").unwrap(), "sst-importer"),
+            (
+                Regex::new(r"^store-read-low\d*$").unwrap(),
+                "store-read-low",
+            ),
+            (Regex::new(r"^rocksdb:bg\d*$").unwrap(), "rocksdb:bg"),
+            (Regex::new(r"^rocksdb:low\d*$").unwrap(), "rocksdb:low"),
+            (Regex::new(r"^rocksdb:high\d*$").unwrap(), "rocksdb:high"),
+            (Regex::new(r"^snap sender\d*$").unwrap(), "snap-sender"),
+            (Regex::new(r"^snap-sender\d*$").unwrap(), "snap-sender"),
+            (Regex::new(r"^apply-\d*$").unwrap(), "apply"),
+            (Regex::new(r"^future-poller-\d*$").unwrap(), "future-poller"),
+        ];
+
+        move |frames| {
+            for (regex, name) in thread_rename.iter() {
+                if regex.is_match(&frames.thread_name) {
+                    frames.thread_name = name.to_string();
```

Comment:
I think the better way is match `^([a-z-]+)\d*` and extract the thread name instead of enumerating all thread names in the current implementation.